monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -34,25 +34,114 @@
-        """
-        An interface for handling `nnU-Net` V2.
-        The users can run the nnU-Net V2 with default settings in one line of code.
-        They can also run parts of the process of nnU-Net V2 instead of the complete pipeline.
+    """
+    ``nnUNetV2Runner`` provides an interface in MONAI to use `nnU-Net` V2 library to analyze, train, and evaluate
+    neural networks for medical image segmentation tasks.
 
-        The output of the interface is a directory that contains:
+    ``nnUNetV2Runner`` can be used in two ways:
 
-            - converted dataset met requirement of nnU-Net V2
-            - data analysis results
-            - checkpoints from the trained U-Net models
-            - validation accuracy in each fold of cross-validation
-            - the predictions on the testing datasets from the final algorithm ensemble and potential post-processing
+    #. with one line of code to execute the complete pipeline.
+    #. with a series of commands to run each modules in the pipeline.
 
-        Args:
-            input_config: the configuration dictionary or the file path to the configuration in form of YAML.
-                The configuration should contain ``datalist``, ``dataroot``, ``modality``.
-            work_dir: working directory to save the intermediate and final results.
-        """
+    The output of the interface is a directory that contains:
+
+    #. converted dataset met the requirement of nnU-Net V2
+    #. data analysis results
+    #. checkpoints from the trained U-Net models
+    #. validation accuracy in each fold of cross-validation
+    #. the predictions on the testing datasets from the final algorithm ensemble and potential post-processing
+
+    Args:
+        input_config: the configuration dictionary or the file path to the configuration in the form of YAML.
+            The keys required in the configuration are:
+            - ``"datalist"``: File path to the datalist for the train/testing splits
+            - ``"dataroot"``: File path to the dataset
+            - ``"modality"``: Imaging modality, e.g. "CT", ["T2", "ADC"]
+            Currently, the configuration supports these optional keys:
+            - ``"nnunet_raw"``: File path that will be written to env variable for nnU-Net
+            - ``"nnunet_preprocessed"``: File path that will be written to env variable for nnU-Net
+            - ``"nnunet_results"``: File path that will be written to env variable for nnU-Net
+            - ``"nnUNet_trained_models"``
+            - ``"dataset_name_or_id"``: Name or Integer ID of the dataset
+            If an optional key is not specified, then the pipeline will use the default values.
+        work_dir: working directory to save the intermediate and final results.
+
+    Examples:
+        - Use the one-liner to start the nnU-Net workflow
+
+        .. code-block:: bash
+
+            python -m monai.apps.nnunet nnUNetV2Runner run --input_config ./input.yaml
```

Comment:
Just a comment that previously these docstrings of examples are in the other file because all of them are bash commands instead of python commands.

Reply:
@wyli thanks for the explanation! Which way would sound better, putting them in the `__main__.py` or the script that defines the class?

Reply:
I'm fine with either, maybe we should add some pythonic examples later, in that case put the docstring here is better